modules/cell-type-ewings/resources/usr/bin/assign-celltypes.R

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env Rscript
+
+# This script is used to assign cell type annotations to Ewing sarcoma libraries in SCPCP000015
+# Tumor cells are assigned based on those with AUC values above specified thresholds for specific gene sets
+# Tumor cells are then classified into EWS-high, EWS-low, and EWS-high proliferative
+# Normal cells are all non-tumor cells and are labeled with the consensus cell type
+
+library(optparse)
+
+option_list <- list(
+  make_option(
+    opt_str = c("--sce_file"),
+    type = "character",
+    help = "Path to RDS file containing a processed SingleCellExperiment object annotate."
+  ),
+  make_option(
+    opt_str = c("--consensus_celltype_file"),
+    type = "character",
+    help = "Path to TSV file containing consensus cell type labels for a single library."
+  ),
+  make_option(
+    opt_str = c("--aucell_results_file"),
+    type = "character",
+    default = NULL,
+    help = "Path to TSV file containing the AUC values for a set of gene sets output by aucell.R"
+  ),
+  make_option(
+    opt_str = c("--auc_table"),
```

Comment:
for consistency, perhaps:
```suggestion
    opt_str = c("--auc_thresholds_file"),
```